docs/adrs/adr-002-deps-mngmt.md

```diff
@@ -0,0 +1,17 @@
+## Context
+
+Ideally all developers working on a project should never fall into the pitfall of “it works on my machine” - the tools they are using, its configuration should be identical all across the systems, resulting in reproducible artifacts and behaviours, from the dev hosts up to “production environments”. 
```

Comment:
Small nitpick here, but it might be better to use ascii quotes "" vs unicode.

Reply:
Fixed, thanks!